### PR TITLE
Add whats new in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/24809

### CODE REVIEW

File: release_notes/whats_new.adoc [hunk 1]
```diff
@@ -31,14 +31,20 @@ Learn more in the link:../console/console.adoc#console-overview[Console overview
 [#cluster-whats-new]
 == Clusters
 
-* The _multicluster engine operator_ is generally available as a software operator that enhances cluster fleet management. The multicluster engine operator supports {ocp} and Kubernetes cluster lifecycle management across clouds and data centers. {ocp} is a prerequisite for the multicluster engine operator. 
+* The _multicluster engine operator_ is generally available as a software operator that enhances cluster fleet management. The multicluster engine operator supports {ocp} and Kubernetes cluster lifecycle management across clouds and data centers. {ocp} is a prerequisite for the multicluster engine operator.
+
+* The permission Cluster set `bind` grants permission to bind the cluster set to a namespace by creating a `ManagedClusterSetBinding`. See link:../multicluster_engine/cluster_lifecycle/managedclustersets_assign_role.adoc##assign-role-clustersets[Assigning users or groups Role-Based Access Control permissions to your ManagedClusterSet] for more details.
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]: I recommend using a lowercase "c" 

The permission Cluster set `bind` grants permission to bind the cluster set to a namespace by creating a `ManagedClusterSetBinding`.

TO

The permission cluster set `bind` grants permission to bind the cluster set to a namespace by creating a `ManagedClusterSetBinding`. 


Also remove the extra hashtag from the link

File: release_notes/whats_new.adoc [hunk 2]
```diff
@@ -31,14 +31,20 @@ Learn more in the link:../console/console.adoc#console-overview[Console overview
...
+
+* When you create a managed cluster, a `ManagedClusterSet` called `global` is automatically created to ease management. You cannot edit the `global` managed cluster set. See link:../multicluster_engine/cluster_lifecycle/managedclustersets_intro.adoc##managedclustersets_global[Global ManagedClusterSet] for more details.
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]: I would delete the following statement: "You cannot edit the `global` managed cluster set." bc the reader will read that in the doc that you reference. In the case that a user is not reading the doc, I wonder there will still be notified about the limitation. Do you know if there will be a note in the console about not being able to edit `global`?

Remove extra hashtag

File: release_notes/whats_new.adoc [hunk 3]
```diff
@@ -31,14 +31,20 @@ Learn more in the link:../console/console.adoc#console-overview[Console overview
...
+
+* You can add multiple hosts to an infrastructure environment at the same time by selecting the **By uploading a YAML** option. See link:../multicluster_engine/cluster_lifecycle/add_hosts_infra_env.adoc##adding-hosts-to-an-infrastructure-environment[Adding hosts to an infrastructure environment] for more details.
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]: Remove the extra hashtag

**Review by dockerymick — CHANGES_REQUESTED** [review]

File: release_notes/whats_new.adoc (hunk 2 above)
Comment (oafischer) [reply release_notes/whats_new.adoc]: The button to edit is just missing in the console for that specific managed cluster set. There isn't anything extra to notify the user about the limitation.

File: release_notes/whats_new.adoc (hunk 3 above)
Comment (oafischer) [reply release_notes/whats_new.adoc]: Whoops, guess I got too carried away with copy pasting the links.

File: release_notes/whats_new.adoc [hunk 4]
```diff
@@ -31,14 +31,20 @@ Learn more in the link:../console/console.adoc#console-overview[Console overview
...
+
+* When you create a managed cluster, a `ManagedClusterSet` called `global` is automatically created to ease management. See link:../multicluster_engine/cluster_lifecycle/managedclustersets_intro.adoc#managedclustersets_global[Global ManagedClusterSet] for more details.
+
+* You can add multiple hosts to an infrastructure environment at the same time by selecting the **By uploading a YAML** option. See link:../multicluster_engine/cluster_lifecycle/add_hosts_infra_env.adoc#adding-hosts-to-an-infrastructure-environment[Adding hosts to an infrastructure environment] for more details.
 
 *Technology Preview:* 
 
 See the following cluster features that are available as Technology Preview:
 
 For other Clusters topics, see link:../clusters/intro.adoc#managing-your-clusters[Managing your clusters].
```
Comment (dockerymick) [inline release_notes/whats_new.adoc:46]: @oafischer Just a heads up, this link is broken. I know that this is due to the MCE changes. 
@swopebe Should this entire link be removed?

**Review by dockerymick — APPROVED** [review]
I made a comment about a link. I plan to start on the link issue next week if you want to move forward with the merge. Wanted to make you and Brandi aware of the link

**Comment by cadawson** [comment]
I think this link is fine. It looks like it should go to the right place when it is live.

**Pull request merged**